### DISCUSSION TITLE
Error when running 'Hello World' application

### DISCUSSION THREAD

**Discussion by pv2-0** [eclipse/mosaic#55] [Q&A] [answered]
I am following the tutorial for building applications. I copied the files [specified here](https://www.eclipse.org/mosaic/docs/develop_applications/) and ran 'mvn clean install' but keep getting the error below:

> [ERROR] Failed to execute goal on project HelloWorldApp: Could not resolve dependencies for project org.eclipse.mosaic.app:HelloWorldApp:jar:0.0.1: Could not find artifact org.eclipse.mosaic:mosaic-application:jar:20.0 in central (https://repo.maven/apache.org/maven2) -> [Help 1]

I followed the link https://repo.maven/apache.org/maven2 and there is no org.eclipse.mosaic:mosaic-application:jar:20.0 there. Based on some searching, I tried running 'mvn clean install -U' but that does not work either.

I would really appreciate any advice.

Thank you

**Comment by kschrab** [comment, marked as answer]
Please add the following to your pom.xml:

```xml
<repositories>
    <repository>
        <id>repo.eclipse.org</id>
        <name>MOSAIC Repository</name>
        <url>https://repo.eclipse.org/content/repositories/mosaic</url>
    </repository>
</repositories>
```

We fix our documentation with the next release.

**Reply by pv2-0** [reply]
That did help, but I am now getting the following error:

> [ERROR] COMPILATION ERROR :
> [INFO] -------------------------------------------------------------
> [ERROR] Source option 5 is no longer supported. Use 6 or later.
> [ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
> [INFO] 2 errors
> [INFO] -------------------------------------------------------------
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time:  34.937 s
> [INFO] Finished at: 2021-03-03T12:31:21-05:00
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project HelloWorldApp: Compilation failure: Compilation failure:
> [ERROR] Source option 5 is no longer supported. Use 6 or later.
> [ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
> [ERROR] -> [Help 1]
> [ERROR]
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR]
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

**Reply by pv2-0** [reply]
I got it to work, just had to add this to pom.xml!

```
<properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
</properties>
```
